distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -960,13 +956,13 @@ type Table
 
        ? `File_Format` write behaviors
 
-         - `File_Format.Auto`: The file format is determined by the file
-           extension of the path argument.
-         - `File_Format.Bytes` and `File_Format.Text`: The Table does not
-           support these types in the `write` function. If passed as format, an
-           `Illegal_Argument_Error` is raised. To write out the table as plain
-           text, the user needs to call the `Text.from Table` method and then
-           use the `Text.write` function.
+         - `Auto_Detect`: The file format is determined by the file extension
+            of the path argument.
```

Comment:
I would not limit ourselves to determining by extension only. Currently that is the case, but I can imagine situations where the format may need to read a few bytes from the file to determine if it can support it.
Do you think such scenarios are feasible in the future? If yes, I'd rephrase this to say just something that the file format is determined automatically based on the provided file?